metrics/cgroups/v2/memory.go

```diff
@@ -0,0 +1,92 @@
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package v2
+
+import (
+	v2 "github.com/containerd/containerd/metrics/types/v2"
+	metrics "github.com/docker/go-metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var memoryMetrics = []*metric{
+	{
+		name: "memory",
```

Comment:
Shouldn't the name be `memory_usage` and the unit be `bytes`? I think you have this and cpu metrics mixed up.  the unit should just be time, bytes, what type of data it is.  the name should be the full `cpu_throttled` and then leave type to the units

Reply:
Oh yeah, i see, will fix it asap